src/engine/src/_main.c

```diff
@@ -390,7 +390,7 @@ NO_OPTIMIZATION void destruct_signal_handling(){
 
 
 NO_OPTIMIZATION void set_thread_params(){
-#if SG_OS != _OS_LINUX
+#if SG_OS != _OS_LINUX || SG_OS != _OS_MAC_OS_X
```

Comment:
I think that should be `&&` instead of `||`, right?

Reply:
Fixed, Should be ok to merge